audio/libsndfile/Portfile

```diff
@@ -1,47 +1,55 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
 PortGroup           github 1.0
 
 github.setup        libsndfile libsndfile 1.0.31
-revision            0
+revision            1
+
 categories          audio
 maintainers         nomaintainer
 platforms           darwin
 
 description         read and write files containing sampled sound
 long_description    \
-	libsndfile is a C library for reading and writing files	\
-	containing sampled sound through one standard library interface. \
-	libsndfile has the following main features: \
-	ability to read and write a large number of file formats, \
-	a simple, elegant and easy to use API, \
-	and on the fly format conversion.
+    libsndfile is a C library for reading and writing files \
+    containing sampled sound through one standard library interface. \
+    libsndfile has the following main features: \
+    ability to read and write a large number of file formats, \
+    a simple, elegant and easy to use API, \
+    and on the fly format conversion.
 license             LGPL-2.1+
+homepage            https://libsndfile.github.io/libsndfile/
+
 checksums           \
-      rmd160  8663673c3f8eaf463c7b499c732c53c9bba03c04 \
-      sha256  f082718e84a27d99c0e349151344b71dc2032e7aa2d799af0da48a8a270aa06a \
-      size    662676
-depends_build-append  port:pkgconfig
-depends_lib           port:flac port:libogg port:libvorbis
+    rmd160  ae3fc5bbcb10a034f3edc1240acacd9f1ec349a7 \
+    sha256  a8cfb1c09ea6e90eff4ca87322d4168cdbe5035cb48717b40bf77e751cc02163 \
+    size    875335
+
+github.tarball_from releases
+master_sites        https://github.com/libsndfile/libsndfile/releases/download/${version}
+distfiles           libsndfile-${version}.tar.bz2
```

Comment:
`distfiles` and `master_sites` should be set automatically by the github PG - did this not work for you (i.e., did you try without these lines)?

Reply:
This was discussed before, the last time I updated this Portfile. The Github PG would just supply it as the Github page, not the actual home page. Or are you saying it would automatically know the link I supplied?

Reply:
I am not sure I understand your comment - the fields I mentioned have nothing to do with setting the homepage, or...?

Reply:
Distfiles is not automatically set. It attempts to fetch a tar.gz of the git repo at the tag, which is not bootstrapped.

Reply:
I see - if you want the `bz2` file instead then you should add `use_bzip2           yes` to the Portfile

Reply:
Oops, I wish I'd read the full documentation, then. Just FYI, there are at least a few Portfiles in the tree that use this same stupid hard coding trick instead of `use_bzip2 yes`. May be worth fixing those as well.